packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -124,4 +129,54 @@ export class PipelineService {
       buttons: [Dialog.okButton()]
     });
   }
+
+  static convertPipeline(pipelineDefinition: any): any {
+    const pipelineJSON = JSON.parse(JSON.stringify(pipelineDefinition));
+
+    if (!Utils.hasPipelineAppdataField(pipelineJSON.pipelines[0], 'version')) {
+      // original pipeline definition without a version
+      console.log(
+        'Migrating pipeline to version: ' + this.CURRENT_PIPELINE_VERSION
```

Comment:
If taking the previous recommendation, this message should indicate both the from and to versions.

Reply:
updated to say `Migrating pipeline to the current version.`